inventory/all_projects/nomad

```diff
@@ -2,32 +2,67 @@
 nomad-host-prod1.lib.princeton.edu
 nomad-host-prod2.lib.princeton.edu
 nomad-host-prod3.lib.princeton.edu
+
 [nomad_clients_production]
 nomad-client-prod1.lib.princeton.edu
 nomad-client-prod2.lib.princeton.edu
 nomad-client-prod3.lib.princeton.edu
 nomad-client-prod4.lib.princeton.edu
+
 [nomad_servers_staging]
 nomad-host-staging1.lib.princeton.edu
 nomad-host-staging2.lib.princeton.edu
 nomad-host-staging3.lib.princeton.edu
+
 [nomad_clients_staging]
 dpul-collections-staging1.lib.princeton.edu
 dpul-collections-staging2.lib.princeton.edu
 nomad-client-staging1.lib.princeton.edu
 nomad-client-staging2.lib.princeton.edu
 nomad-client-staging3.lib.princeton.edu
+
+[nomad_servers_sandbox]
+nomad-host-sandbox1.lib.princeton.edu
+nomad-host-sandbox2.lib.princeton.edu
+nomad-host-sandbox3.lib.princeton.edu
+
+[nomad_clients_sandbox]
+nomad-client-sandbox1.lib.princeton.edu
+nomad-client-sandbox2.lib.princeton.edu
+
 [nomad_clients:children]
 nomad_clients_staging
 nomad_clients_production
+nomad_clients_sandbox
```

Comment:
I don't think this should be here - right now `nomad_clients` is effectively `nomad_clients_production_cluster` or something